cosmos/runtime/miner/miner.go

```diff
@@ -23,19 +23,27 @@ package miner
 
 import (
 	"context"
+	"time"
 
 	"github.com/cosmos/gogoproto/proto"
 
 	"github.com/berachain/polaris/eth"
 
 	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/telemetry"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/ethereum/go-ethereum/common"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/miner"
 )
 
+const (
+	MetricKeyBuildBlock   = "polaris_miner_build_block"
+	MetricKeyBlockGasUsed = "polaris_miner_block_gas_used"
+	MetricKeyTransactions = "polaris_miner_transactions"
```

Comment:
can you throw all the keys into a telemetry vs having them dispersed across the repo